harvester/harvest.py

```diff
@@ -438,9 +438,12 @@ def send_notification_emails(self, results: dict) -> None:
                     server.sendmail(SMTP_CONFIG["default_sender"], [recipient],
                                     msg.as_string())
                     logger.info(f"Notification email sent to: {recipient}")
+                    return "Emails sent successfully"
```

Comment:
the function return str now, so the function hint `-> None` needs to be `-> str`

Reply:
fixed.

Reply:
It feels like it should return None. I don't see the returned string consumed any where.

Reply:
right, just fixed.